### PR TITLE
Highlight shapes with thickening borders

### PR DESCRIPTION
**Pull request by CubePresser**
* Thickened borders overall
* Currently selected shape has thickest border of all shapes
* Hovered cells have a slightly thicker than normal border

### CODE REVIEW

File: src/ColorShape.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
+const DEFAULT_THICKNESS = 4;
+
 export default class ColorShape extends createjs.Shape {
     constructor(color, vertices, neighbors) {
         super();
         this.neighbors = neighbors;
         this.vertices = vertices;
         this.color = color;
+        this.thickness = 4;
```
Comment (Manason) [inline src/ColorShape.js]: shouldn't this be set to DEFAULT_THICKNESS rather than hardcoded?

File: src/ColorShape.js [hunk 2]
```diff
@@ -20,17 +23,18 @@ export default class ColorShape extends createjs.Shape {
...
     changeColor(color) {
-        this.drawSelf(color);
         this.color = color;
     }
 
-    resetColor() {
-        this.drawSelf(this.color);
+    // Permanently change the stroke thickness
+    changeStrokeThickness(thickness = DEFAULT_THICKNESS) {
+        this.thickness = thickness;
```
Comment (Manason) [inline src/ColorShape.js:31]: prefer the convention set and get for field getters and setters i.e. setStrokeThickness()

File: src/Game.js [hunk 3]
```diff
@@ -4,6 +4,9 @@ import ColorShape from './ColorShape';
 import ColorUtil from './util/ColorUtil';
 import ColorQueue from './ColorQueue';
 
+const SELECT_THICKNESS = 6;
+const CURRENT_THICKNESS = 8;
```
Comment (Manason) [inline src/Game.js]: CURRENT THICKNESS is 8 but default is 4? What does current thickness mean? Consider renaming.

File: src/Game.js [hunk 4]
```diff
@@ -48,7 +54,8 @@ export default class Game {
     onMouseOut({ target }) {
         if (!this.validShape(target)) return;
 
-        target.resetColor();
+        target.changeStrokeThickness()
```
Comment (Manason) [inline src/Game.js]: change to what? Consider a separate function to reset stroke thickness to default

Comment (CubePresser) [reply src/Game.js]: By default the thickness parameter here will be DEFAULT_THICKNESS. Its a way of overloading in javascript by setting default parameter values.

**Pull request merged**